### PR TITLE
DROID-418 : Update FoodSafe integrated product values to match the latest changes

### PR DESCRIPTION
**Pull request by jhavatar**
https://linear.app/combustion/issue/DROID-418/update-foods-integrated-mode-parameters-in-ble-framework

### Description
Update Foodsafe Integrated product values to match the latest changes made in PR https://github.com/combustion-inc/combustion-android-ble/pull/125

### Verify values match
Screenshot 2024-06-21 at 21 53 26

### CODE REVIEW

File: meatnet_node_ble_specification.rst [hunk 1]
```diff
@@ -1124,23 +1124,20 @@ sync purposes. The values are interpreted exclusively by the client in
 Integrated mode; the firmware performs the food safety calculations based on
 the other values supplied.
```
Comment (jmaha) [inline meatnet_node_ble_specification.rst:1126]: Please add a note to these lists in both documents explaining that missing values are deprecated, to guide future maintainers.

**Review by jmaha — APPROVED** [review]
Looks good! One small comment.

File: meatnet_node_ble_specification.rst (hunk 1 above)
Comment (jhavatar) [reply meatnet_node_ble_specification.rst:1128]: @jmaha added a note - what you expected?

Comment (jmaha) [reply meatnet_node_ble_specification.rst:1128]: That works, thanks!

**Review by jmaha — APPROVED** [review]
Looks great!

**Pull request merged**